deployment/aks-periscope.yaml

```
@@ -143,10 +143,10 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: clustertype-config
+  name: flag-config
   namespace: aks-periscope
 data:
-  CLUSTER_TYPE: "managedCluster" # <custom flag>
+  FLAG_LIST: # <custom flag> connectedCluster OSM
```

```suggestion
  FLAG_LIST: # <custom flag> "connectedCluster" "OSM"
```
does this need to be a string in quotes based on the prev example? shouldn't matter right? I think yaml should be able to render it?

---

@SanyaKochhar Yeah it doesn't matter if the string is wrapped in quotes or not the yaml can detect that it's a string